packages/mesa/install_megadrivers.py.patch

```diff
@@ -0,0 +1,11 @@
+--- src/bin/install_megadrivers.py	2023-02-09 00:05:36.000000000 +0300
++++ src/bin/install_megadrivers.py.patch	2023-02-16 12:31:41.280068558 +0300
+@@ -55,7 +55,7 @@
+         if os.path.lexists(abs_driver):
+             os.unlink(abs_driver)
+         print('installing {} to {}'.format(args.megadriver, abs_driver))
+-        os.link(master, abs_driver)
++        os.symlink(master, abs_driver)
+ 
+         try:
+             ret = os.getcwd()
```

Comment:
https://github.com/termux/termux-packages/pull/15225#discussion_r1109669514 also applies to this patch.

Reply:
Logical